staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -198,26 +198,25 @@ const (
 // AdmissionHookClientConfig contains the information to make a TLS
 // connection with the webhook
 type AdmissionHookClientConfig struct {
-	// Service is a reference to the service for this webhook. If there is only
-	// one port open for the service, that port will be used. If there are multiple
-	// ports open, port 443 will be used if it is open, otherwise it is an error.
-	// Required
-	Service ServiceReference `json:"service" protobuf:"bytes,1,opt,name=service"`
-
-	// URLPath is an optional field that specifies the URL path to use when posting the AdmissionReview object.
-	URLPath string `json:"urlPath" protobuf:"bytes,3,opt,name=urlPath"`
+	// `url` gives the location of the webhook, in standard URL form
+	// (`[scheme://]host:port/path`).
+	//
+	// If the host refers to a service running in the cluster, it must be
+	// of the form `servicename.namespace.svc.`. Do not include the
+	// "cluster.local" suffix, as that is a configurable detail and would
+	// render your config non-portable; apiservers will route to the
+	// service correctly.
+	//
+	// If the scheme is present, it must be "https://".
```

Comment:
Why isn't http:// permissible?

Reply:
If it were, then the CABundle field would have to be optional, but it isn't.

I don't mind changing this--was just keeping the change limited.